src/App.vue

```diff
@@ -24,14 +20,13 @@ export default {
   provide() {
     return {
       savedParks: this.savedParks,
-      savePark: this.savePark
     }
   },
-  methods: {
-    savePark(newPark) {
-      this.savedParks = [...this.savedParks, newPark] 
-    }
-  }
+  created() {
+    eventBus.$on('savePark', (data) => {
+      this.savedParks.push(data);
+    })
+  },
 }
```

Comment:
Do we want to add something here to prevent duplicates from being added to the savedParks?

Reply:
Good call, Stacy! I'll add an issue to our project board for that.